.github/workflows/tests.yaml

```diff
@@ -153,6 +153,13 @@ jobs:
           github.ref == 'refs/heads/main' && github.event_name == 'push' }}" >>
           "$GITHUB_OUTPUT"
 
+      - id: devcontainer-build
+        run:
+          echo "run=${{ steps.devcontainer-push.outputs.run == 'true' ||
+          steps.changes.outputs.devcontainer-build == 'true' ||
+          steps.changes.outputs.nightly-schedule == 'true' }}" >>
+          "$GITHUB_OUTPUT"
```

Comment:
Perhaps this is related to the fact that GitHub Actions have a bool type but Shell does not.
Are you sure that choosing `'true'` instead of `true` here is correct?

Reply:
Here, inside `${{ }}` IIUC _everything_ is stringified. So I think we always want `== 'true'`...